src/actions/profileActions.js

```diff
@@ -0,0 +1,80 @@
+import axios from '../config/axiosConfig';
+import { GET_PROFILE, PROFILE_LOADING, UPDATE_PROFILE } from '../action-types';
+
+// export const {
```

Comment:
@samorahno  Looks like you have commented out the code here. You can remove it since you won't be needing it

Reply:
@Chiazokam thanks, this has been fixed